test/setupAfterEnv.ts

```diff
@@ -0,0 +1,8 @@
+import { Node1Client, SDKClient } from '../src/thor-client'
+
+jest.retryTimes(3, { logErrorsBeforeRetry: false })
+
+afterAll(async () => {
```

Comment:
what is the diff between afterall and globalteardown ? would not be safer to run all the after stuff in teardown ? not sure how it works with tests running in parallel

Reply:
So if we're running different workers (ours is defaulting to 3), it can have open connections in each worker, this script runs in each work, after the tests run.

globalteardown runs after all of the workers have finished, it could in theory be used to clean up files, tear down docker compose etc.

I will remove `globalTeardown.ts`

Reply:
Yeah I think cleanup for each worker should be enough especially if it runs after they have finished